prediction/tools/model_warmup/README.md

```diff
@@ -0,0 +1,129 @@
+# Model warmup
+
+If you come a cross a familiar message:
+
+```
+2020–01–23 23:58:40.355818: I tensorflow_serving/servables/tensorflow/saved_model_warmup.cc:105] No warmup data file found at /models/resnet/1538687457/assets.extra/tf_serving_warmup_requests
+```
+
+Note: This document can be used in AI Platform prediction during model creation.
+
+From TensorFlow Serving documentation:
+
+```
+The TensorFlow runtime has components that are lazily initialized, which can cause high latency for the first request/s sent to a model after it is loaded. This latency can be several orders of magnitude higher than that of a single inference request.
+To reduce the impact of lazy initialization on request latency, it’s possible to trigger the initialization of the sub-systems and components at model load time by providing a sample set of inference requests along with the SavedModel. This process is known as “warming up” the model.
+```
+
+I’m using `ResNet50v2` model for CPU
+
+Model location [here](https://github.com/tensorflow/models/tree/master/official/r1/resnet) 
+
+```
+mkdir /tmp/resnet
+curl -s http://download.tensorflow.org/models/official/20181001_resnet/savedmodels/resnet_v2_fp32_savedmodel_NHWC_jpg.tar.gz | tar — strip-components=2 -C /tmp/resnet -xvz
+ls /tmp/resnet/*
+saved_model.pb variables
+```
+
+Verify model using `saved_model_cli` utility:
```

Comment:
Please number all the steps so it's easier for readers to tell which actions they have to take.
You can put "1. " in front of each step, and then Markdown should render the numbering correctly.

Reply:
Thanks Crystal